### PR TITLE
464 adding indexes into ddl model

### PR DESCRIPTION
**Pull request by LukeLoze1**
# Has the DDL schema changed?
*Pull requests which include schema changes should be labelled with the `schema change` label for visibility*

*The [common entity model pom.xml](ssdc-rm-common-entity-model/pom.xml) file `project.version` must be bumped
appropriately if there are any common entity code changes*

* [x] This PR includes DDL schema changes and has been labelled correctly, bumping to new schema
  version: 
* [ ] The POM has been updated with an appropriate version bump if required

# Motivation and Context
New indexes and run the indexes on the 

# What has changed
added new patch
new file for running in GIN indexes, now run on Groundzero

# How to test?
Check patch 1st
Then run in the ddl groundzero in the env of your choice.  Has it created the indexes expected?

# Links
https://trello.com/b/VS0ZyRmj/srm

### CODE REVIEW

**Review by AdamHawtin — CHANGES_REQUESTED** [review]
The uac_qid_link qid index needs to be added to the common entity model.

**Comment by LukeLoze1** [comment]
> The uac_qid_link qid index needs to be added to the common entity model.

File: ssdc-rm-common-entity-model/src/main/java/uk/gov/ons/ssdc/common/model/entity/UacQidLink.java [hunk 1]
```diff
@@ -19,6 +19,9 @@
 @TypeDefs({@TypeDef(name = "jsonb", typeClass = JsonBinaryType.class)})
 @Entity
 @DynamicUpdate
+@Table(
+    name = "uac_qid_link",
+    indexes = {@Index(name = "uac_qid_idx", columnList = "qid")})
```
Comment (AdamHawtin) [inline ssdc-rm-common-entity-model/src/main/java/uk/gov/ons/ssdc/common/model/entity/UacQidLink.java:24]: Small point but a more precise name on the index might be helpful, `uac_qid_idx` isn't clear which column it refers to

**Review by AdamHawtin — APPROVED** [review]
LGTM 👍

**Review by ryangrundy7 — APPROVED** [review]

**Pull request merged**